app/serializers/submission_serializer/definitions/partner.rb

```diff
@@ -29,9 +29,19 @@ def to_builder
           json.involvement_in_case partner_detail.involvement_in_case
           json.conflict_of_interest partner_detail.conflict_of_interest
           json.has_same_address_as_client partner_detail.has_same_address_as_client
+
+          json.is_included_in_means_assessment partner_in_means_assessment?
```

Comment:
Is there a way to include the type of means assessment module in this class? I'm getting an error
`undefined local variable or method `crime_application'...`

Reply:
This was the same issue I had when I added the module to a decision tree https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/pull/917/files#r1635155390

Just define crime_application for the module to use @hiboabd